packages/rbac/index.ts

```diff
@@ -1,136 +1,136 @@
 // This is a sample supporting file mustache template.
 
-export * from './types'
```

Comment:
Something will need to happen to the `mustache` template for this top level `index.ts` as we won't get the paths for `v1` endpoint (and soon v2) automatically anymore when generating